abis_mapping/templates/survey_metadata/templates/instructions.md

```diff
@@ -0,0 +1,143 @@
+# SYSTEMATIC SURVEY METADATA TEMPLATE INSTRUCTIONS
+
+## OVERVIEW
+Use this template to record systematic survey metadata; that is metadata relating to a
+Systematic Survey dataset.
+
+This Systematic Survey Metadata template must be used in combination with the Systematic
+Survey Occurrence template and, in some cases, the Systematic Survey Site template.
```

Comment:
"Survey Occurrence template and, in some cases, the Systematic Survey Site template." -> "Survey Occurrence template and in some cases, the Systematic Survey Site template." ?